content/en/docs/migration/faq-migration.md

```diff
@@ -94,6 +50,39 @@ In an announcement in August 2019, the expectation was set that the Flux project
 
 After experimentation and considerable thought, we (the maintainers) have found a path to Flux v2 that we think better serves our vision of GitOps: the GitOps Toolkit. In consequence, we do not now plan to integrate GitOps Engine into Flux.
 
+## Helm Operator and Helm Controller
+
+### Are automated image updates supported?
+
+Not yet, but the feature is under active development. See the [image update feature parity section on the roadmap](/docs/roadmap/#flux-image-update-feature-parity) for updates on this topic.
+
+### How do I automatically apply my `HelmRelease` resources to the cluster?
+
+If you are currently a Flux v1 user, you can commit the `HelmRelease` resources to Git, and Flux will automatically apply them to the cluster like any other resource. It does however not support automated image updates for Helm Controller resources.
+
+If you are not a Flux v1 user or want to fully migrate to Flux v2, the [Kustomize Controller](/docs/components/kustomize/controller/) will serve your needs.
+
+### I am still running Helm v2, what is the right upgrade path for me?
+
+Migrate your Helm v2 releases to v3 using [the Helm Operator's migration feature](/legacy/helm-operator/helmrelease-guide/release-configuration/#migrating-from-helm-v2-to-v3), or make use of the [`helm-2to3`](https://github.com/helm/helm-2to3) plugin directly, before continuing following the [migration steps](#steps).
+
+### Is the Helm Controller ready for production?
+
+Probably, but with some side notes:
+
+1. It is still under active development, and while our focus has been to stabilize the API as much as we can during the first development phase, we do not guarantee there will not be any breaking changes before we reach General Availability. We are however committed to provide [conversion webhooks](https://kubernetes.io/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definition-versioning/#webhook-conversion) for upcoming API versions.
```

Comment:
Aren't we guaranteeing now that APIs will stay stable? https://fluxcd.io/docs/components/helm/api/